### DISCUSSION TITLE
Tours built in

### DISCUSSION THREAD

**Discussion by PeteDuncanson** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#21] [General]
Should have a custom tour that shows you around and walks you through/encourages you to add a property to an existing doctype, edit the template to render it and then preview it etc. for all the main things you might want to do. Been an obvious quick win for years but never seems to have been done.

**Comment by melvinvanrookhuizen** [comment]
For me the tours are a tool aimed at the end usr, the editor. Umbrqco should come wit a full set of tours for editors.(willing to contribute)

**Comment by PeteDuncanson** [comment]
Absolutely, when you go to edit the content in the starter kit it can walk you through that. Bigger question though...what is the starter kit for? Spinning up a real site you can build on or a quick way to show you what you “could do” and how best to do it?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by melvinvanrookhuizen** [comment]
That is a good question indeed. It touches on a bigger problem that has been there for a long time. What part of communication and documentation is pointed at the end-user and waht part to developers. This also goes for the starterkit. Now most of the default content and documentation is aimed at developers. I think there should be completely different sets. A demo starterkit for end users (content managers), with the available documentation for editors setup in Tours. With the dashboard filled with content aimed at that group of users and not aimed at developers.
And same goes for Umbraco TV, there are no official sets for the end users about umbraco basics.




Melvin van Rookhuizen
<https://nl.linkedin.com/in/joycedatema>https://nl.linkedin.com/in/melvinvanrookhuizen
06-15909546
 [1479189872460_handtekening]
Maliebaan 24-26
3581CB  Utrecht
www.zaaks.nl<http://www.zaaks.nl>
030-2769146


________________________________
Van: Pete Duncanson <notifications@github.com>
Verzonden: vrijdag 25 september 2020 13:01
Aan: TimGeyssens/Umbraco-v8-Community-Website-StarterKit <Umbraco-v8-Community-Website-StarterKit@noreply.github.com>
CC: Melvin van Rookhuizen <melvin@zaaks.nl>; Comment <comment@noreply.github.com>
Onderwerp: Re: [TimGeyssens/Umbraco-v8-Community-Website-StarterKit] Tours built in (#21)


Absolutely, when you go to edit the content in the starter kit it can walk you through that. Bigger question though...what is the starter kit for? Spinning up a real site you can build on or a quick way to show you what you “could do” and how best to do it?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by PeteDuncanson** [comment]
Wow, we ended up at the deadend of “trying to fix the business rather than build a starter kit” real fast there! 😊

HQ should be trying to grow there business via their marketing department who should be chipping in on ideas like you raise but that takes time and effort and through put.

I’d suggest focusing on the narrow window of what can be influenced rather than trying to over-reach and “fix all the things” because they are “totally broken™”.

I would like to state that I agree with everything you raise, all very valid but also immoveable from this side of the wall.

**Comment by melvinvanrookhuizen** [comment]
Yes, I agreed. Its a bigger problem than you raise. (and a favourite of mine as you might notice ;-) )

Ok, uhm. Let's try and give the starterkit(s) more approriate names? So there will be 'Developer Starter Kits' and 'Demo starter kits?'' So innocent shy editors looking for een Umbraco demo or instruction dont get sucked into =obvously brilliant= code or tech mumbo jumbo ?

**Reply by melvinvanrookhuizen** [reply]
ow, to go back to your initial rquestion: yes, tours should be built in from the start as an encouragement for developers.

**Comment by TimGeyssens** [comment]
I think we should make a real site, showcasing the best practises... will it end up as a started site ... maybe ... but mainly for inspiration , ideas on the goal welcome here #8

**Comment by TimGeyssens** [comment]
but yeah tours would be nice, if they can also be dev orientated
